packages/core/src/content/content-client.ts

```diff
@@ -100,4 +106,29 @@ export class ContentClient {
 
     return this.graphqlClient.request<T>(query, variables, options);
   }
+
+  /**
+   * Retrieves the locale information for a given locale ID.
+   *
+   * @param id - The unique identifier of the locale item.
+   * @returns A promise that resolves to the locale information associated with the specified locale ID.
+   */
+  async getLocale(id: string) {
```

Comment:
This method can return `null` when locale with the `id` doesn't exist, please update method return type to have null as well